lib/tasks/interactive-new.js

```diff
@@ -121,6 +121,22 @@ class InteractiveNewTask extends Task {
           },
         ],
       },
+      {
+        name: 'emberData',
+        type: 'confirm',
+        message: 'Do you want to include ember-data?',
+        when: undefined === newCommandOptions.emberData,
+        choices: [
```

Comment:
These had to go.

Reply:
https://github.com/ember-cli/ember-cli/pull/10622